pachyderm/templates/etcd/amazon-storageclass.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
   name: etcd-storage-class
   namespace: default
 parameters:
-  type: gp2
+  type: gp3
```

Comment:
I like this change, but am hesitant to ship it without testing, also we should consider the upgrade case

(Ditto for the reclaim policy)

Reply:
I'm never against testing, but I'm not sure what the intent is here. Is there a problem with using gp3 as a storage class?

Reply:
On reclaim policy: This is already a recommendation we push to customers. We're simply including something we already recommend in the base configuration. (This will only be relevant for 1.x pachyderm anyway.)

Reply:
I took out reclaim policy for now, we can discuss in more detail later